index.js

```diff
@@ -167,6 +188,39 @@ let handleTrackName = async (channel) => {
     }
 }
 
+let handleQueue = async (channel) => {
+    try {
+        await printQueue(channel);
+    } catch (error) {
+        // Token expired
+        if(error?.response?.data?.error?.status === 401) {
+            await refreshAccessToken();
+            await printQueue(channel);
+        } else {
+            client.say(chatbotConfig.channel_name, `Seems like no music is playing right now`);
+        }
+    }
+}
+
+let handleVoteSkip = async (channel, username) => {
+
+    if (!usersHaveSkipped.has(username)) {
+        startOrProgressVoteskip(channel);
+
+        usersHaveSkipped.add(username);
+        console.log(`${username} voted to skip the current song (${usersHaveSkipped.size}/${chatbotConfig.required_vote_skip})!`);
+        client.say(channel, `${username} voted to skip the current song (${usersHaveSkipped.size}/${chatbotConfig.required_vote_skip})!`);
```

Comment:
This should probably be DM'd to the user or at most only said once ("username has started a vote to skip!"), otherwise it will probably lead to a decent amount of spam. Also preferably configurable